### ISSUE TITLE
Fix description of initial balloon position

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#221]
The balloon is initially described as "In upper right side of play area..."  It should be "In center of play area, evenly between sweater and wall."

From #215

**Comment by jessegreenberg** [comment]
This is no longer a problem, on load balloon is "At center of play area".
